README.md

```diff
@@ -68,14 +81,34 @@ Below you'll find an image that illustrates the different AACT databases and sch
   
     `source ~/.zshrc` to load the variables into the terminal session.  
     
-    Depending on where you store the variables you may need to call `source` on that file each time you open a new terminal. This is not necessary for “.zshrc”.   
+    Depending on where you store the variables you may need to call `source` on that file each time you open a new terminal. This is not necessary for “.zshrc”.
+     When you done check your “.zshrc” or ".bashrs" file. It should looks like example below.If it doesn't - edit it accordingly. 
+
+    Example of a set of environment variables: 
+
+export AACT_PUBLIC_BETA_DATABASE_NAME=aact
+export AACT_PUBLIC_DATABASE_NAME=aact
+export AACT_ALT_PUBLIC_DATABASE_NAME=aact_alt
+export AACT_PASSWORD=54104754
+export AACT_USERNAME=user_name
+export PGPASSWORD=passw
+export TEST_PUBLIC_DB_USER=user_name
+export TEST_PUBLIC_DB_PASS=passw
+export PUBLIC_DB_USER=user_name
+export PUBLIC_DB_PASS=passw
+export AACT_DB_SUPER_USERNAME=user_name
+export PATH=$PATH:/lib/postgresql/15/bin
+export AACT_CORE_DATABASE_URL=postgres://DB_USER_NAME:DB_PASSW@localhost:5432/aact
+export AACT_CORE_TEST_DATABASE_URL=postgres://DB_USER_NAME:DB_PASSW@127.0.0.1:5432/aact_test
+# Add RVM to PATH for scripting. Make sure this is the last PATH variable change.
+export PATH="$PATH:$HOME/.rvm/bin"  
 
 5.  Clone this repo: `git clone git@github.com:ctti-clinicaltrials/aact.git`  
     Note: Cloning with a ssh url requires that your local ssh key is saved to Github. The key verifies your permission to push and pull so you won't have to log in. If you haven't saved your ssh key to Github use the html url instead when cloning.  
 
 6.  `cd aact` to enter the directory  
 
-7.  Install a ruby version manager like rbenv, then install Ruby 2.6.2  
+7.  Install a ruby version manager like rbenv, then install Ruby 2.7.7  
```

Comment:
Should we add a commands here for windows | linux | macOs?

Reply:
For Ruby installation and project cloning? I don't think so, people may google it. I do that every year slightly differently - information quickly becomes outdated.